templates/jslaves.yml.j2

```diff
@@ -1,8 +1,14 @@
 ---
 jslaves:
 {% for item in jslave_list|select("match", ".*benchmark.*") %}
-	- name: "{{ item }}"
-	  cgroup:
+	- name: {{ item }}
+{% if item is match(".*qdc.*") %}
+          cgroup: qualcomm
+{% elif item is match(".*d0.*") %}
+	  cgroup: huawei
+{% elif item is match(".*tx2.*") %}
```

Comment:
tx1 would be the Thunderx1, if we end up having some. "tx.*" probably better, here.